modules/selenium/src/test/java/org/testcontainers/junit/ChromeRecordingWebDriverContainerTest.java

```diff
@@ -59,6 +66,53 @@ public String getFilesystemFriendlyName() {
                 assertEquals("Recorded file not found", 1, files.length);
             }
         }
+
+        @Test
+        public void recordingTestThatShouldHaveCorrectDuration() throws IOException, InterruptedException {
+            File target = vncRecordingDirectory.getRoot();
+            try (
+                // recordAll {
+                // To do this, simply add extra parameters to the rule constructor:
+                BrowserWebDriverContainer<?> chrome = new BrowserWebDriverContainer<>()
+                    .withCapabilities(new ChromeOptions())
+                    .withRecordingMode(RECORD_ALL, target)
+                    .withRecordingFileFactory(new DefaultRecordingFileFactory())
+                    .withNetwork(NETWORK)
+            ) {
+                chrome.start();
+
+                TimeUnit.SECONDS.sleep(5);
+                doSimpleExplore(chrome);
+                chrome.afterTest(new TestDescription() {
+                    @Override
+                    public String getTestId() {
+                        return getFilesystemFriendlyName();
+                    }
+
+                    @Override
+                    public String getFilesystemFriendlyName() {
+                        return "ChromeThatRecordsAllTests-recordingTestThatShouldBeRecordedAndRetained";
+                    }
+                }, Optional.empty());
+
+                String flvFileTitle = "ChromeThatRecordsAllTests-recordingTestThatShouldBeRecordedAndRetained";
+                String flvFileNameRegEx = "PASSED-" + flvFileTitle + ".*\\.flv";
+                String recordedFile = vncRecordingDirectory.getRoot().listFiles(new PatternFilenameFilter(flvFileNameRegEx))[0].getCanonicalPath();
+
+                try( GenericContainer<?> container = new GenericContainer<>(TestcontainersConfiguration.getInstance().getVncDockerImageName()) ) {
+                    String recordFileContainerPath = "/tmp/chromeTestRecord.flv";
+                    container.withCopyFileToContainer(MountableFile.forHostPath(recordedFile), recordFileContainerPath)
+                            .withCreateContainerCmdModifier( createContainerCmd -> createContainerCmd.withEntrypoint("ffmpeg") )
+                            .withStartupTimeout( Duration.ofMinutes(1) )
+                            .withCommand("-i" , recordFileContainerPath, "-f" ,"null" ,"-" )
+                            .start();
+                    String ffmpegOutput = container.getLogs();
```

Comment:
I think it is not guaranteed, that the command did finish, therefore you miss the logs in CI, which runs slower. This container is missing the appropriate Wait Strategy. `LogMessageWaitStrategy` seems to be a good candidate here.

Reply:
done, thanks for your help